antsibull/cli/antsibull_build.py

```diff
@@ -101,6 +101,13 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
     common_parser.add_argument('--dest-dir', default='.',
                                help='Directory to write the output to')
 
+    cache_parser = argparse.ArgumentParser(add_help=False)
+    cache_parser.add_argument('--collection-cache', default=None,
+                              help='Directory of cached collection tarballs.  Will be'
+                              ' used if a collection tarball to be downloaded exists'
+                              ' in here, and will be populated when downloading new'
+                              ' tarballs.')
+
```

Comment:
Why not make this part of the build_parser?

Reply:
Because in #103 I'm not using the `build_parser`, and I need this option separately.

Reply:
Okay, cool.  Makes sense to me.